inc/dbmysql.class.php

```diff
@@ -57,42 +52,33 @@ class DBmysql {
    //! Database Error
    public $error              = 0;
 
-   /// Slave management
+   // Slave management
    public $slave              = false;
    /** Is it a first connection ?
     * Indicates if the first connection attempt is successful or not
     * if first attempt fail -> display a warning which indicates that glpi is in readonly
    **/
    public $first_connection   = true;
-   /// Is connected to the DB ?
+   // Is connected to the DB ?
    public $connected          = false;
 
 
    /**
     * Constructor / Connect to the MySQL Database
     *
-    * try to connect
-    *
-    * @param $choice integer, host number (default NULL)
-    *
-    * @return nothing
```

Comment:
Yes, I agree this is not mandatory. But I do not know if that can be excluded from CS doc checks :/